asciidocs/setup.adoc

```diff
@@ -2,19 +2,19 @@
 
 There are two main ways to get started with Seqera's Nextflow training course.
 
-The first is to install the requirements locally (section 1.1), which is best if you are already familiar with Git and Docker, or working offline.
+The first is to install the requirements locally (<<Local installation>>), which is best if you are already familiar with Git and Docker, or working offline.
 
-The second is to use Gitpod (section 1.2), which is best for first-timers as this platform contains all the programs and data required.
-Simply click the link and login via your GitHub account to start the tutorial.
+The second is to use Gitpod (<<Gitpod>>), which is best for first-timers as this platform contains all the programs and data required.
+Simply click the link and log in using your GitHub account to start the tutorial.
 
 == Local installation
 
-Nextflow can be used on any POSIX-compatible system (Linux, OS X, Windows Subsystem for Linux, etc.).
+Nextflow can be used on any POSIX-compatible system (Linux, macOS, Windows Subsystem for Linux, etc.).
 
 *Requirements*:
 
 * Bash
-* http://www.oracle.com/technetwork/java/javase/downloads/index.html[Java 8 (or later, up to 15)]
+* https://www.oracle.com/technetwork/java/javase/downloads/index.html[Java 8 (or later, up to 15)]
```

Comment:
Actually, here it's best to recommend Java-11 as the minimal version similar to the official NF docs.

Tho worth keeping in mind that we only have automated testing with Java-11/17

```suggestion
* https://www.oracle.com/technetwork/java/javase/downloads/index.html[Java 11 (or later, up to 18)]
```

Reply:
Thanks for the suggestions @abhi18av, this adds colour for me. I was just editing the content directly, without referencing up-to-date info. In the sections to come, I'll make sure versioning and other info is current.